packages/client/hmi-client/src/components/articles/Document.vue

```diff
@@ -206,6 +211,19 @@ const openPDF = () => {
 	window.open(docLink.value as string);
 };
 
+/**
+ * Format from xDD citation_list object.
+ *
+ * -  author (year), title, journal, doi
+ */
+const formatCitation = (obj: { [key: string]: string }) => {
+	if (Object.keys(obj).length <= 1) {
+		if (obj.unstructured_citation) return obj.unstructured_citation;
+		return '';
+	}
+	return `${obj.author}, ${obj.year}, "${obj.title}", ${obj.journal}, ${obj.doi}`;
+};
```

Comment:
This should be a Document Util function.

Reply:
Will clean up in #496